### ISSUE TITLE
Cant install (This is serious)

### ISSUE THREAD

**Issue by Luis3819** [LoukaMB/SynapseX#149]
Hello.
My name is Luis.
Im having a trouble to install synapse x, it has a virus, but its not a false possitive.
The virus that it have is a backdoor called bladabindi, you can search it on YouTube.
Im really scared of getting a virus on my pc.
Please, i need help. 
Its serious.
Luis.

**Comment by IsntLovely** [comment]
Yes is does have the bladabindi virus wither its fake or real i have no idea better safe than sorry. Uninstall till fix i guess
